### PR TITLE
Centralize `Background.md` files

### PR DESCRIPTION
**Pull request by kabilar**
- Move `Background.md` files from each `element` repository to this central location so that it is visible on the https://elements.datajoint.org website.
- Update the path to the `Projects` and `Publications` page.
- Update README to include a table:
image

Future work after this pull request is merged:
- Remove the `Background.md` files from the respective `element` repositories and update the `README` files to point to the correct location for the background documentation.
- Update the contents of the background files.

### CODE REVIEW

**Comment by kabilar** [comment]
Updated the screenshot in the comment above to reflect the most recent commits.

File: gh-pages/docs/description/miniscope.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Miniscope Element
```
Comment (CBroz1) [inline gh-pages/docs/description/miniscope.md:1]:
```suggestion
# Miniscope
```

File: gh-pages/docs/description/calcium_imaging.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Calcium Imaging Element
```
Comment (CBroz1) [inline gh-pages/docs/description/calcium_imaging.md:1]:
```suggestion
# Calcium Imaging
```

File: gh-pages/docs/description/array_ephys.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Array Electrophysiology Element
```
Comment (CBroz1) [inline gh-pages/docs/description/array_ephys.md:1]: It looked odd to me that the drop-down was inconsistent in the use of `Element` - So I removed from respective titles. Will that be reflected in the dropdown menu when rendered?
```suggestion
# Array Electrophysiology
```

File: README.md [hunk 4]
```diff
@@ -32,22 +32,25 @@ Elements are described in the position paper
...
+| [Animal management](https://github.com/datajoint/element-animal) | See workflow for session management | [Background, features, and status](description/animal.md) |
+| [Experiment session management](https://github.com/datajoint/element-session) | [Workflow for lab, subject and session management](https://github.com/datajoint/workflow-session) | [Background, features, and status](description/session.md) |
+|  [Extracellular array electrophysiology](https://github.com/datajoint/element-array-ephys) | [Workflow for Neuropixels](https://github.com/datajoint/workflow-array-ephys) | [Background, features, and status](description/array_electrophysiology.md) |
+|  [Calcium imaging](https://github.com/datajoint/element-calcium-imaging) | [Workflow for multiphoton laser scanning](https://github.com/datajoint/workflow-calcium-imaging) | [Background, features, and status](description/calcium_imaging.md) |
+|  [Miniscope imaging](https://github.com/datajoint/element-miniscope) | [Workflow for UCLA Miniscope](https://github.com/datajoint/workflow-miniscope) | [Background, features, and status](description/miniscope.md) |
```
Comment (CBroz1) [inline README.md:43]: A more condensed version

```suggestion
| DataJoint Element | Example workflow | Backgound, Features, and Status |
|:-|:-|:-|
| [Lab management](https://github.com/datajoint/element-lab) | See Session | [Link](description/lab.md) |
| [Animal management](https://github.com/datajoint/element-animal) | See Session | [Link](description/animal.md) |
| [Experiment Session management](https://github.com/datajoint/element-session) | [Lab, subject and session management](https://github.com/datajoint/workflow-session) | [Link](description/session.md) |
| [Extracellular array electrophysiology](https://github.com/datajoint/element-array-ephys) | [Neuropixels](https://github.com/datajoint/workflow-array-ephys) | [Link](description/array_electrophysiology.md) |
| [Calcium imaging](https://github.com/datajoint/element-calcium-imaging) | [Multiphoton laser scanning](https://github.com/datajoint/workflow-calcium-imaging) | [Link](description/calcium_imaging.md) |
| [Miniscope imaging](https://github.com/datajoint/element-miniscope) | [UCLA Miniscope](https://github.com/datajoint/workflow-miniscope) | [Link](description/miniscope.md) |
```

**Review by CBroz1 — APPROVED** [review]

File: gh-pages/docs/description/array_ephys.md (hunk 3 above)
Comment (kabilar) [reply gh-pages/docs/description/array_ephys.md]: Thanks @CBroz1.  These changes should be reflected in the dropdown menu.

**Review by CBroz1 — APPROVED** [review]

**Pull request merged**